CHANGELOG.md

```diff
@@ -46,199 +54,198 @@ This file is used to list changes made in each version of the splunk cookbook.
 - Improved guards to prevent `service[splunk]` restart/start when it should be disabled.
 
 ## 6.3.0 (2020-10-14)
+
 - Fixes Issue [#183](https://github.com/chef-cookbooks/chef-splunk/issues/183): make upgrades idempotent
 - it is no longer necessary to include `chef-splunk::upgrade` to a run list; Instead, set the following:
-  - set `node['splunk']['server']['upgrade']['version']` or `node['splunk']['forwarder']['upgrade']['version']` for the appropriate server type
-  - set `node['splunk']['server']['upgrade']['url']` or `node['splunk']['forwarder']['upgrade']['url']` for the appropriate server type
-  - set `node['splunk']['upgrade_enabled'] = true`
+- set `node['splunk']['server']['upgrade']['version']` or `node['splunk']['forwarder']['upgrade']['version']` for the appropriate server type
+- set `node['splunk']['server']['upgrade']['url']` or `node['splunk']['forwarder']['upgrade']['url']` for the appropriate server type
+- set `node['splunk']['upgrade_enabled'] = true`
 
 ## 6.2.11 (2020-10-14)
+
 - Sets the Splunk Enterprise Server and Forwarder upgrade versions to 8.0.6
 - Sets the upgrade attributes to pull v8.0.6 of Splunk Enterprise and Universal Forwarder
 
 ## 6.2.10 (2020-07-15)
+
 - Fixes Issue [#178](https://github.com/chef-cookbooks/chef-splunk/issues/178)
 - `#shcluster_member?` passes `node['ipaddress']` to `#include?`
 
 ## 6.2.9 (2020-07-01)
+
 - Drops travis-ci and onboards testing with Github Actions
 - removes dependency on the splunk command when running the `disable` recipe
 
 ## 6.2.8 (2020-07-01)
+
 - Thank you, [@doublethink](https://github.com/doublethink), for this submission
-- Resolves issues installing server and client as non-root users:
-  * `chef-splunk::user` recipe will not run if splunkd should be run as a non-root user
-  * systemd and SysV templates correctly run as the specified non-root user
```

Comment:
Sub-lists should be supported as long as they are formatted properly. Can you please fix this so that it still retains this but is correct according to `mdl`?